src/vaadin-pdf-viewer.html

```diff
@@ -81,10 +83,45 @@
             /**
              * A represenentation of a document that has been read in. 
              */
-            _document: Object
+            _document: Object,
```

Comment:
Probably it's a good idea to make all of those properties private, not protected.
https://polymer-library.polymer-project.org/3.0/docs/devguide/properties#private-and-protected-properties

Reply:
Done. I made them __viewer, __document, __title, __filename and __pdfTitle